github-markdown.css

```diff
@@ -138,10 +133,8 @@
   color: #032f62;
 }
 
-.markdown-body .octicon {
-  display: inline-block;
-  vertical-align: text-top;
-  fill: currentColor;
+.markdown-body .placeholder-box {
```

Comment:
I don't think this one is relevant to markdown.

Reply:
You're right, but it gets past `uncss` somehow.  Investigating.

Reply:
I ended up having to explicitly exclude it.